doc/hma-ui.md

```diff
@@ -0,0 +1,280 @@
+# Introduction to Hasher-Matcher-Actioner UI
+
+Open Media Match (aka Hasher-Matcher-Actioner) includes a web app where users can configure their hash banks, connect to hash exchanges like NCMEC, Stop NCII, GIFCT, and ThreatExchange, and debug matches. These features are available as demos to show the features and functionality of HMA.  
+
+There are 4 pages in the app:
+
+## **Dashboard (`/ui/`)**
+
```

Comment:
Ignorable/Idea: Include a link to the default localhost port hosted version for each of these pages.